packages/salesforcedx-vscode-apex/src/commands/forceLaunchApexReplayDebuggerWithCurrentFile.ts

```diff
@@ -10,53 +10,65 @@ import {
   CommandBuilder,
   EmptyParametersGatherer,
   fileExtensionsMatch,
-  fileUtils,
   notificationService,
   SfdxCommandlet,
   SfdxCommandletExecutor,
   SfdxWorkspaceChecker
 } from '@salesforce/salesforcedx-utils-vscode';
 import * as vscode from 'vscode';
+import { channelService } from '../channels';
 import { nls } from '../messages';
 import { testOutlineProvider } from '../views/testOutlineProvider';
 import { forceAnonApexDebug } from './forceAnonApexExecute';
 
 export async function forceLaunchApexReplayDebuggerWithCurrentFile() {
-  const editor = vscode.window.activeTextEditor;
-  if (!editor) {
+  try {
+    const editor = vscode.window.activeTextEditor;
+    if (!editor) {
+      notificationService.showErrorMessage(
+        nls.localize('unable_to_locate_editor')
+      );
+      return;
+    }
+
+    const sourceUri = editor.document.uri;
+    if (!sourceUri) {
+      notificationService.showErrorMessage(
+        nls.localize('unable_to_locate_document')
+      );
+      return;
+    }
+
+    if (isLogFile(sourceUri)) {
+      await launchReplayDebuggerLogFile(sourceUri);
+      return;
+    }
+
+    if (isAnonymousApexFile(sourceUri)) {
+      await launchAnonymousApexReplayDebugger();
+      return;
+    }
+
+    const apexTestClassName = await getApexTestClassName(sourceUri);
+    if (apexTestClassName) {
+      await launchApexReplayDebugger(apexTestClassName);
+      return;
+    }
+
     notificationService.showErrorMessage(
-      nls.localize('unable_to_locate_editor')
+      nls.localize('launch_apex_replay_debugger_unsupported_file')
     );
-    return;
-  }
-
-  const sourceUri = editor.document.uri;
-  if (!sourceUri) {
+  } catch (error) {
     notificationService.showErrorMessage(
-      nls.localize('unable_to_locate_document')
+      nls.localize('unknown_error_when_launching_apex_replay_debugger')
     );
-    return;
-  }
-
-  if (isLogFile(sourceUri)) {
-    await launchReplayDebuggerLogFile(sourceUri);
-    return;
-  }
 
-  if (isAnonymousApexFile(sourceUri)) {
-    await launchAnonymousApexReplayDebugger();
-    return;
+    channelService.showChannelOutput();
```

Comment:
In the `catch` block, a notification and logging to the Output view were added.  A new string was added and the verbiage was created by Sonal.